.travis.yml

```diff
@@ -69,9 +70,11 @@ script:
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then bundle exec bundle-audit check --update --ignore CVE-2015-9284; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then yarn build-storybook; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "2" ]; then bin/test-console-check; fi
-  - export FOREM_OWNER_SECRET="secret"
+  - export FOREM_OWNER_SECRET="secret" # test secret so e2e tests can run properly.
+  - rake data_updates:run # required for at least creating the first admin user in e2e tests.
   - yarn e2e:ci
   # Dropping and recreating the database for Capybara/rspecs
+  - curl -XDELETE "${ELASTICSEARCH_URL}/*" # resetting elastic search post e2e tests
```

Comment:
I had to drop all the indices because there was data populated in here from the e2e tests that were breaking Capybara/rspec tests.

Reply:
🔥 no leftovers!